osc-bsu-csi-driver/values.yaml

```diff
@@ -34,29 +34,31 @@ backoff:
 sidecars:
   provisionerImage:
     repository: registry.k8s.io/sig-storage/csi-provisioner
-    tag: "v3.0.0"
+    tag: "v5.1.0"
     # -- Enable http endpoint to get metrics of the container
     enableHttpEndpoint: false
     # -- Port of the http endpoint
     httpEndpointPort: "8089"
     # -- Enable liveness probe for the container
     enableLivenessProbe: false
     # -- Customize leaderElection, you can specify `leaseDuration`, `renewDeadline` and/or `retryPeriod`. Each value must be in an acceptable time.ParseDuration format.(Ref: https://pkg.go.dev/flag#Duration)
+    additionalArgs: []
+    # Grant additional permissions to external-provisioner
+    additionalClusterRoleRules:
```

Comment:
default value  '{}' ?

Reply:
A list ([]) allows additionalArgs to accurately reflect the nature of command-line flags and parameters, which need to be handled as a sequence of values rather than as independent key-value pairs.
When generating templates or validating configurations, it’s much simpler to ensure that each item is a string and can be concatenated properly to form the full command.